docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -206,3 +207,9 @@ export AZURE_ACCOUNT_NAME='test'
 export AZURE_ACCOUNT_ACCESS_KEY='NN1KHxKKkj20ZO92EMiDQjx3wp2kZG4UUvfAGlgGWRn6sPRmGY/TEST/Dri+ExAmPlEExAmPlExA+ExAmPlExA=='
 ```
 ````
+
+## Databricks File System (DBFS)
+
```

Comment:
Can you also add a section `MosaicML platform` and link [this](https://docs.mosaicml.com/projects/mcli/en/latest/resources/secrets/env.html) env doc and explain how user can set the env variable (`DATABRICKS_HOST` and `DATABRICKS_TOKEN`) using mcli command.